src/odemis/gui/comp/overlay/world_select.py

```diff
@@ -239,3 +242,268 @@ def on_motion(self, evt):
             WorldOverlay.on_motion(self, evt)
 
     # END Event Handlers
+
+
+class RectanglePointsSelectOverlay(WorldOverlay, RectangleMixin):
```

Comment:
Can you add a docstring to this class to explain what it does, or why you'd want to use it?